src/design-system/color/palettes.ts

```diff
@@ -505,7 +505,7 @@ export const foregroundColors: Record<
   },
   'buttonStrokeSecondary': {
     light: globalColors.white20,
-    dark: globalColors.white20,
+    dark: globalColors.grey20,
```

Comment:
i believe the original color here was incorrect. it's not used anywhere outside of this pr, so there are no side effects